pom.xml

```diff
@@ -0,0 +1,385 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.carbon.esb.connector</groupId>
+    <artifactId>org.wso2.carbon.esb.connector.amazonlambda</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <name>WSO2 Carbon - Connector For Amazon Lambda</name>
+    <url>http://wso2.org</url>
+
+    <properties>
+        <connector.name>amazonlambda</connector.name>
+        <integration.base.version>1.0.4</integration.base.version>
+        <product.ei.version>6.4.0</product.ei.version>
+        <carbon.kernel.version>4.4.17</carbon.kernel.version>
+        <automation.framework.utils.version>4.4.2</automation.framework.utils.version>
+        <automation.framework.version>4.4.3</automation.framework.version>
+        <emma.version>2.1.5320</emma.version>
+        <synapse.version>2.1.7-wso2v19</synapse.version>
+        <carbon.mediation.version>4.6.19</carbon.mediation.version>
+        <wso2.json.version>2.0.0.wso2v1</wso2.json.version>
+        <org.testng.version>6.1.1</org.testng.version>
+        <jets3t.version>0.9.4</jets3t.version>
+        <bouncycastle.version>1.55</bouncycastle.version>
+        <rampart.apache.version>1.6.1-wso2v18</rampart.apache.version>
+        <rampart.wso2.version>1.6.1.wso2v18</rampart.wso2.version>
+        <httpmime.version>4.3.6</httpmime.version>
+        <json.version>20080701</json.version>
+        <skip-tests>false</skip-tests>
```

Comment:
By default, we need to skip the test (ie when we perform the mvn clean install) Hence please make it as true.

Reply:
Fix with 2f7700e